package.json

```diff
@@ -39,11 +39,12 @@
   "scripts": {
     "spin": "nodemon --watch content --watch static --watch templates --watch spin-up-hub --ext html,md,rhai,hbs,css,js --verbose --legacy-watch --signal SIGINT --exec 'npm run build-index && npm run build-hub-index && spin up --file spin.toml --quiet --env PREVIEW_MODE=$PREVIEW_MODE'",
     "bundle-scripts": "parcel watch static/js/src/main.js --dist-dir static/js --no-source-maps",
+    "bundle-scripts-ci": "parcel build static/js/src/main.js --dist-dir static/js --no-source-maps",
```

Comment:
minor nit: since we've added this to the spin.toml as an integral build step for a component, should this just be `bundle-scripts` and the other one `bundle-scripts-watch` or other?  No strong opinion, feel free to veto :)

Reply:
I am indifferent. I just choose to add a new script so as to not disturb the existing workflow.

Reply:
Or well I cannot merge with failing CI tests since I do not have the perms.